docs/tempo/website/getting-started/tempo-in-grafana.md

```diff
@@ -47,6 +47,16 @@ The Tempo configuration is the same for searching recent traces or
 for search of the backend datastore. 
 -  Run Grafana 8.3.6 or a more recent version. Enable the `tempoBackendSearch` [feature toggle](https://github.com/grafana/tempo/blob/main/example/docker-compose/tempo-search/grafana.ini). This will cause Grafana to pass the `start` and `end` parameters necessary for the backend datastore search.
 
+## APM dashboard
+
+Grafana provides a built-in Application Performance Management (APM) dashboard.
```

Comment:
should we mention the Grafana version this is available in?

Reply:
Sure! Which version should be listed? Is it also available in Grafana Cloud?

Reply:
9.1 :) If Grafana Cloud is running 9.1 then it is available.

Reply:
Added Grafana 9.1 and Grafana Cloud to the line above.